diff --git a/package/boot/uboot-mediatek/Makefile b/package/boot/uboot-mediatek/Makefile
--- a/package/boot/uboot-mediatek/Makefile
+++ b/package/boot/uboot-mediatek/Makefile
@@ -859,7 +859,7 @@ endef
 define U-Boot/mt7986_netcore_n60-pro
   NAME:=Netcore N60 Pro
   BUILD_SUBTARGET:=filogic
-  BUILD_DEVICES:=netcore_n60-pro
+  BUILD_DEVICES:=netcore_n60-pro netcore_n60-pro-512rom
   UBOOT_CONFIG:=mt7986_netcore_n60-pro
   UBOOT_IMAGE:=u-boot.fip
   BL2_BOOTDEV:=spim-nand

diff --git a/package/boot/uboot-tools/uboot-envtools/files/mediatek_filogic b/package/boot/uboot-tools/uboot-envtools/files/mediatek_filogic
--- a/package/boot/uboot-tools/uboot-envtools/files/mediatek_filogic
+++ b/package/boot/uboot-tools/uboot-envtools/files/mediatek_filogic
@@ -41,6 +41,7 @@ konka,komi-a31|\
 mercusys,mr90x-v1-ubi|\
 netcore,n60|\
 netcore,n60-pro|\
+netcore,n60-pro-512rom|\
 netis,eap930-v1|\
 netis,nx30v2|\
 netis,nx31|\
 netis,nx32u|\
